dmutils/apiclient/data.py

```diff
@@ -197,6 +197,15 @@ def register_framework_agreement_returned(self, supplier_id, framework_slug, use
             },
         )
 
+    def find_framework_suppliers(self, framework_slug, agreement_returned=None):
+        params = {}
+        if agreement_returned is not None:
+            params['agreement_returned'] = 'true' if agreement_returned else 'false'
```

Comment:
It can't default to `agreement_return=False` because that would only return supplier frameworks that have not had an agreement returned. The default is to return all supplier frameworks for this framework. The cast to the string would work though.